src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTestBase.java

```diff
@@ -81,15 +80,21 @@ ProducerService producerService() {
     }
 
     @BeforeAll
-    static void beforeAll() {
+    static void beforeAll(VertxTestContext context) {
         kafkaCluster.start();
         vertx = Vertx.vertx();
 
-        if (!"TRUE".equalsIgnoreCase(System.getenv("STRIMZI_USE_SYSTEM_BRIDGE"))) {
+        LOGGER.info("Environment variable " + " EXTERNAL_BRIDGE:" + BRIDGE_EXTERNAL_ENV);
+
+        if ("FALSE".equals(BRIDGE_EXTERNAL_ENV)) {
             bridgeConfig = BridgeConfig.fromMap(config);
             httpBridge = new HttpBridge(bridgeConfig);
             httpBridge.setHealthChecker(new HealthChecker());
+
+            LOGGER.info("Deploying in-memory bridge");
+            vertx.deployVerticle(httpBridge, context.succeeding(id -> context.completeNow()));
         }
+        // else we create external bridge from the OS invoked by `.jar`
```

Comment:
is this a TODO?

Reply:
No, it is not. We will start .jar in the pipeline before tests will be triggered.